.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug report
```

Comment:
ISSUE: Inconsistent capitalization.

We use a capital H for Help, but use lowercase R for report and request.

- Need Help?
- Bug report
- Feature request

We should be consistent.